src/components/VersionSelect.tsx

```diff
@@ -38,6 +39,8 @@ interface Props {
 
 export default function VersionSelect({ isLoadingSwc }: Props) {
   const [swcVersion, setSwcVersion] = useAtom(swcVersionAtom)
+  const [deferedSwcVersion, setDeferedSwcVersion] = useState(swcVersion)
+  const [isPending, startTransition] = useTransition()
```

Comment:
Using `useTransition` for switching SWC version is unnecessary, I think, which makes code more complex. Or, using Suspense may be considerable.

Reply:
Ok :) , I think using `useDeferredValue` would help us from writing duplicate code :)